### DISCUSSION TITLE
Radiation subroutine

### DISCUSSION THREAD

**Discussion by lihy213** [firemodels/fds#13901] [Q&A]
Hello team,
I still work on sodium fire. Recently I want to add the radiation properties of sodium and its oxides to RCALLOC. In RCALLOC subroutine which belongs to `rcal.f90`, there're already data for several species, so is it necessary to add the radiation properties of sodium to it? Or is it better to use the default data of some defined species? If I want to add the radiation properties of sodium, where could I find these kinds of BAND data?
Kind regards.
LI

**Comment by drjfloyd** [comment]
Radiation properties are species dependent.  You would need to locate detailed spectral data for your species of interest in the literature or make your own measurements.

**Reply by lihy213** [reply]
Thanks, but in which reference you find these data? Like here below the BAND for METHANE. In the source file, it doesn't annotate it.
![图片](https://github.com/user-attachments/assets/57a38aa0-9ba8-408b-ac71-04b8ee874e5b)

**Reply by drjfloyd** [reply]
You will need to search the literature (there are many web based search engines for scientific publications) or make your own measurements.

**Reply by lihy213** [reply]
Fine! I'll try it.
